core/storage/user/gae_models.py

```diff
@@ -357,11 +380,15 @@ def get_by_email(cls, email):
             UserSettingsModel | None. The UserSettingsModel instance which
             contains the same email.
         """
-        filtered_users = cls.query(cls.email == email).fetch()
+        filtered_users = cast(
+            List[UserSettingsModel],
+            cls.query(cls.email == email).fetch()
+        )
```

Comment:
Hmm, can we use `get` instead of fetch here? (since we are returning only one result anyway

Reply:
Yes, changed!